about/system_requirements.rst

```diff
@@ -23,7 +23,7 @@ Desktop or laptop PC - Minimum
 .. which can run up to macOS 10.13.
 
 +----------------------+-----------------------------------------------------------------------------------------+
-| **CPU**              | - **Windows:** x86_32 CPU with SSE2 instructions, or any x86_64 CPU                     |
+| **CPU**              | - **Windows:** x86_32 CPU with SSE2 instructions, x86_64 CPU, ARMv8 CPU                 |
 |                      |                                                                                         |
 |                      |   - *Example: Intel Core 2 Duo E8200, AMD Athlon XE BE-2300*                            |
```

Comment:
For the reference, it is possible to run ARM64 Windows version Godot at least on some Snapdragon 8xx devices, but I'm not sure if it should be listed as minimum (low performance, lack of testing, and potential issues with old GPU drivers), we probably should assume Snapdragon X Elite as minimum.

Reply:
Missed that one, good catch! Just updated the PR.